### ISSUE TITLE
Better CLI output

### ISSUE THREAD

**Issue by rrdelaney** [rrdelaney/ReasonablyTyped#35]
This is different than #34 because it focuses on the good part, successful output!

Right now the output is pretty simple, only shows what compiled to what and how many files were compiled:

<img width="1102" alt="screen shot 2017-08-14 at 10 53 16 pm" src="https://user-images.githubusercontent.com/6325382/29303689-70be34e4-8143-11e7-83dc-7c2d28cc914c.png">

It would be nice to have some check marks or maybe emoji output for fun 😄 Maybe compilation time for each file in the output too! This is pretty open-ended, and up for discussion!

### Implementation details

Basically all code for this will be in [the CLI interface](https://github.com/rrdelaney/ReasonablyTyped/blob/master/lib/cli.js). No Reason experience required!
